gnosis/eth/utils.py

```diff
@@ -20,13 +21,25 @@ def get_empty_tx_params() -> TxParams:
     }
 
 
+@lru_cache(maxsize=8192)
 def fast_keccak(value: bytes) -> Hash32:
     """
     Calculates ethereum keccak256 using fast library `pysha3`
+
+    :param value:
+    :return: Keccak256 used by ethereum as `HexBytes`
+    """
+    return HexBytes(keccak_256(value).digest())
+
+
+def fast_keccak_text(value: str) -> Hash32:
```

Comment:
```suggestion
def fast_keccak_text(value: str) -> HexBytes:
```
It is returning an object not string.

Reply:
`Hash32` is a `bytes`, and `HexBytes` is a sublass of `bytes`